README.md

```diff
@@ -72,6 +72,8 @@ Flags:
       --shutdown-grace-period=1m
                          Wait this long for sessions to end before shutting
                          down.
+      --email-domain     Restrict access to users with emails that match
+                         this domain
```

Comment:
A nitpick, but I'd prefer this help output be created by running `kuberos --help` and pasting it in. It's already a bit stale so you'd be doing me a favour. :)